gptme/llm/llm_anthropic.py

```diff
@@ -224,7 +303,7 @@ def _transform_system_messages(
 
     # for any subsequent system messages, transform them into a <system> message
     for i, message in enumerate(messages):
-        if message.role == "system":
+        if message.role == "system" and message.call_id is None:
```

Comment:
We don't want to change messages with call_id. They will be handled by the `_handle_tool` function.

Reply:
Done.